src/patternfly/components/Popover/popover.scss

```diff
@@ -67,38 +94,90 @@
     .pf-c-popover__arrow {
       bottom: 0;
       left: 50%;
-      transform: translateX(var(--pf-c-popover__arrow--m-top--TranslateX)) translateY(var(--pf-c-popover__arrow--m-top--TranslateY)) rotate(var(--pf-c-popover__arrow--m-top--Rotate));
+      transform:
+        translateX(var(--pf-c-popover__arrow--m-top--TranslateX))
+        translateY(var(--pf-c-popover__arrow--m-top--TranslateY)) rotate(var(--pf-c-popover__arrow--m-top--Rotate));
     }
   }
 
   &.pf-m-bottom {
     .pf-c-popover__arrow {
       top: 0;
       left: 50%;
-      transform: translateX(var(--pf-c-popover__arrow--m-bottom--TranslateX)) translateY(var(--pf-c-popover__arrow--m-bottom--TranslateY)) rotate(var(--pf-c-popover__arrow--m-bottom--Rotate));
+      transform:
+        translateX(var(--pf-c-popover__arrow--m-bottom--TranslateX))
+        translateY(var(--pf-c-popover__arrow--m-bottom--TranslateY))
+        rotate(var(--pf-c-popover__arrow--m-bottom--Rotate));
     }
   }
 
   &.pf-m-left {
     .pf-c-popover__arrow {
       top: 50%;
       right: 0;
-      transform: translateX(var(--pf-c-popover__arrow--m-left--TranslateX)) translateY(var(--pf-c-popover__arrow--m-left--TranslateY)) rotate(var(--pf-c-popover__arrow--m-left--Rotate));
+      transform:
+        translateX(var(--pf-c-popover__arrow--m-left--TranslateX))
+        translateY(var(--pf-c-popover__arrow--m-left--TranslateY)) rotate(var(--pf-c-popover__arrow--m-left--Rotate));
     }
   }
 
   &.pf-m-right {
     .pf-c-popover__arrow {
       top: 50%;
       left: 0;
-      transform: translateX(var(--pf-c-popover__arrow--m-right--TranslateX)) translateY(var(--pf-c-popover__arrow--m-right--TranslateY)) rotate(var(--pf-c-popover__arrow--m-right--Rotate));
+      transform:
+        translateX(var(--pf-c-popover__arrow--m-right--TranslateX))
+        translateY(var(--pf-c-popover__arrow--m-right--TranslateY)) rotate(var(--pf-c-popover__arrow--m-right--Rotate));
+    }
+  }
+
+  &.pf-m-success {
+    --pf-c-popover__icon--Color: var(--pf-c-popover--m-success__icon--Color);
+
+    .pf-c-title {
+      margin-left: var(--pf-global--spacer--xl);
+    }
+  }
+
+  &.pf-m-error {
+    --pf-c-popover__icon--Color: var(--pf-c-popover--m-error__icon--Color);
+
+    .pf-c-title {
+      margin-left: var(--pf-global--spacer--xl);
     }
   }
+
+  &.pf-m-warning {
+    --pf-c-popover__icon--Color: var(--pf-c-popover--m-warning__icon--Color);
+
+    .pf-c-title {
+      margin-left: var(--pf-global--spacer--xl);
+    }
+  }
+
+  &.pf-m-info {
+    --pf-c-popover__icon--Color: var(--pf-c-popover--m-info__icon--Color);
+
+    .pf-c-title {
+      margin-left: var(--pf-global--spacer--xl);
+    }
+  }
+
+  .pf-c-popover__icon {
+    position: absolute;
```

Comment:
using `position: absolute` doesn't scale very well since it takes the element out of the document flow. For example, if a user introduces a larger image here than the one we've accounted for, they will also need to update the title's `margin-left` to account for the image size/width.

You should just be able to use regular positioning, and remove this and the `right` properties if you move the icon in the title, and make the title a flex layout.

Reply:
This is not applicable anymore as I used modalBox component to reimplement the popover.